include/gsl/span

```diff
@@ -785,65 +814,74 @@ constexpr span<typename Ptr::element_type> make_span(Ptr& cont)
 }
 
 // Specialization of gsl::at for span
-template <class ElementType, std::ptrdiff_t Extent>
+template <class ElementType, std::size_t Extent>
 constexpr ElementType& at(span<ElementType, Extent> s, index i)
 {
     // No bounds checking here because it is done in span::operator[] called below
     return s[i];
 }
 
 // [span.obs] Free observer functions
-template <class ElementType, std::ptrdiff_t Extent>
-constexpr typename span<ElementType, Extent>::index_type ssize(const span<ElementType, Extent> &span) noexcept
+template <class ElementType, std::size_t Extent>
+constexpr typename span<ElementType, Extent>::index_type
+ssize(const span<ElementType, Extent>& span) noexcept
```

Comment:
`ssize` is now returning an unsigned `index_type` which is contrary to the intent of the function.